Description: Fix manpage section
 Manual pages generated by pod2man are installed in section 1, we want them
 installed in section .1p because they are Perl-specific.
 And the manpage for bin/nytprofmerge is missing.
Forwarded: no
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2010-06-12

--- a/Makefile.PL
+++ b/Makefile.PL
@@ -29,9 +29,10 @@
     @man = ( MAN1PODS => {}, MAN3PODS => {} );
 } else {
     @man = ( MAN1PODS  => {
-        'bin/nytprofhtml' => '$(INST_MAN1DIR)/nytprofhtml.1',
-        'bin/nytprofcsv'  => '$(INST_MAN1DIR)/nytprofcsv.1',
-        'bin/nytprofcg'   => '$(INST_MAN1DIR)/nytprofcg.1',
+        'bin/nytprofhtml' => '$(INST_MAN1DIR)/nytprofhtml.1p',
+        'bin/nytprofcsv'  => '$(INST_MAN1DIR)/nytprofcsv.1p',
+        'bin/nytprofcg'   => '$(INST_MAN1DIR)/nytprofcg.1p',
+        'bin/nytprofmerge'=> '$(INST_MAN1DIR)/nytprofmerge.1p',
     } );
 }
 
